apps/keyboard/js/imes/myanmar/myanmar.js

```diff
@@ -0,0 +1,249 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Please help make sure you run jshint against this file to ensure it can pass the lint check. Thanks.